viral/rastermap_utils.py

```
@@ -375,57 +374,63 @@ def process_trials_data(
         valid_mask = filter_speed_position(
             speed=speed,
             frames_positions=frames_positions,
-            speed_threshold=0.5,
-            position_threshold=180,
-            filter_speed=False,
-            filter_position=False,
         )
         if np.all(valid_mask == False):
```

Ruff (0.8.2)

Avoid equality comparisons to `False`; use `if not valid_mask:` for false checks

Replace with `not valid_mask`

(E712)